提交分析pcap包实验

xaZKX opened the pull request.

c4pr1c3 commented:
Accepted with minor revision

* 「开启网卡的监听模式」小节缺失图片 `wlan0mon_monitor.png`
* 笔误：查看**以**保存的文件 -> **已**
* tshark 输出 SSID 里的乱码、「空白」有思路如何实现自动化解码（非现在的手工逐个解码）？
* 「同名SSID只取一个」？那如果是不同 SSID 但 AP 开启了「隐藏 SSID」功能的情况你的这个统计方法会有什么问题吗？
* 「判断是否加密根据wlan.fixed.capabilities.privacy判断，0为非加密，1为加密」，那么，你的实验报告中给出的 `360WiFi` 这个字段为空该如何理解呢？提示：你想一想，哪些类型的帧头包含 wlan.fixed.capabilities.privacy 这个字段？
* 「加密方式」的判断有问题，需要继续查阅资料或其他同学作业或我们的课本学习了解；
* 推荐阅读目前为止我已经批改完的优秀同学作业点评
    * https://github.com/CUCCS/2018-NS-Public-TheMasterOfMagic/pull/3
    * https://github.com/CUCCS/2018-NS-Public-jckling/pull/3

xaZKX commented:
> 「开启网卡的监听模式」小节缺失图片 wlan0mon_monitor.png
- 已补上

> 笔误：查看以保存的文件 -> 已
- 已改正

> tshark 输出 SSID 里的乱码、「空白」有思路如何实现自动化解码（非现在的手工逐个解码）？
- 查看了优秀作业中关于scapy自动化解决乱码的问题。

> 「同名SSID只取一个」？那如果是不同 SSID 但 AP 开启了「隐藏 SSID」功能的情况你的这个统计方法会有什么问题吗
- 同名SSID只取一个，不同 SSID 但 AP 开启了「隐藏 SSID」功能如果去重就会被当成一个，可能会造成SSID统计缺少问题。 应该可以用 cat -v 的方式区分「一串0x00」与「真的什么也没有」，然后在进行计算。

> 「判断是否加密根据wlan.fixed.capabilities.privacy判断，0为非加密，1为加密」，那么，你的实验报告中给出的 360WiFi 这个字段为空该如何理解呢？提示：你想一想，哪些类型的帧头包含 wlan.fixed.capabilities.privacy 这个字段？
- Beacon帧和Pro Response帧头包含wlan.fixed.capabilities.privacy 这个字段，360WiFi这个字段为空是因为它的帧类型为Pro Request，所以这个判断依据不准确。
